assets/src/apps/v2/triptych.tsx

```diff
@@ -125,7 +127,7 @@ const App = (): JSX.Element => {
         <Route exact path="/v2/screen/:id/simulation">
           <MappingContext.Provider value={TYPE_TO_COMPONENT}>
             <ResponseMapperContext.Provider value={responseMapper}>
-              <SimulationScreenPage />
+              <SimulationScreenPage opts={{ alternateView: true }} />
```

Comment:
Just making sure I understand when we want this opt, since I previously removed it for the triptych app (oops)—is the alternate view for any screen type that doesn't have any kind of "flex zone"/paged region? Seems that way since this is also used for DUPs.

Reply:
The opt is to indicate that a separate component will handle the rendering of the simulation. For most of our screens, we are ok rendering the full page as a single image. For DUPs, we needed something special in order to meet design requirements. That's why DUPs use the option. 

Now that I'm thinking about it, Triptychs do not need this option right now because we are ok shipping a super simple view of the screen. I added the option for those because I anticipated a special design, but that's not in scope right now. Gonna just use the standard template for now and we can add a custom template in the future if needed.